### DISCUSSION TITLE
Accessing the Spinx roles in Jupyter Books

### DISCUSSION THREAD

**Discussion by DamoM73** [talkpython/static-sites-with-sphinx-and-markdown-course#11] [Q&A] [answered]
Hi just finished the course and found it very helpful.

I'm a secondary teacher, and I've been using Jupyter Books to create tutorial sites from my students. I know that Jupyter Books uses Sphix, and I would love to access some the Sphinx functions from within Jupyter Books (especially the file download link). Can do this?

**Comment by pauleveritt** [comment, marked as answer]
Alas, I don't have any experience with Jupyter Books, but the Executable Books folks have been super-helpful over the years.

**Reply by DamoM73** [reply]
Thanks for you quick reply. I've tried a different tact. I'm creating the site using Sphinx, MyST and your tutorial, and then I'm using the Executable Book Project Theme.

**Comment by DamoM73** [comment]
One more question. I would like to create pds of my lessons. Can you recommend how to go about this using Sphix on Windows?
